Analyticsv3Sample/AnalyticsSampleProject/Program.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DAIMTO.Analytics.v3.Sample
+{
+    /// <summary>
+    /// View and manage your Google Analytics data
+    /// https://developers.google.com/analytics/
+    /// In order to run this you must install the nuget package.  Please consult the readme.
+    /// 
+    /// Link to Nuget Package: https://www.nuget.org/packages/Google.Apis.analytics.v3/
+    /// PM> Install-Package Google.Apis.Analytics.v3 
+    /// </summary>
+    class Program
+    {
+        static void Main(string[] args)
+        {
+
```

Comment:
* What should we include here?   
* Should we give an example of authentication?   (OAuth2, service account, API Key or all three)
* Should we give an example of using the methods and looping though the results?